pkg/scheduler/objects/queue_test.go

```diff
@@ -28,7 +28,7 @@ import (
 
 	"gotest.tools/v3/assert"
 
-	"github.com/apache/yunikorn-core/pkg/common"
+	pkg_common "github.com/apache/yunikorn-core/pkg/common"
```

Comment:
We have another common from SI, but that's referenced as `siCommon`, is this necessary?

Reply:
Yes, this is not needed. Thanks.